v3.1/how-to/upgrade/from-previous-version.md

```diff
@@ -6,7 +6,7 @@ category: how-to
 
 # TiDB 3.0 Upgrade Guide
 
-This document is targeted for users who want to upgrade from TiDB 2.0 (2.0.1 or later versions) or TiDB 2.1 RC to TiDB 3.0. TiDB 3.0 is compatible with [TiDB Binlog of Kafka Version](/v3.1/reference/tidb-binlog/tidb-binlog-kafka.md) and [TiDB Binlog of Cluster Version](/v3.1/reference/tidb-binlog/overview.md).
+This document is targeted for users who want to upgrade from TiDB 2.0 (2.0.1 or later versions) or TiDB 2.1 (2.1 RC1 or later versions) to TiDB 3.0 or the master version. Currently, TiDB 3.0 and the master version are compatible with [TiDB Binlog of Cluster Version](/v3.1/reference/tidb-binlog/overview.md).
```

Comment:
@liubo0127 @ericsyh Please take a look at the v3.1 upgrade guide. If only minor updates are needed, we can update it in this PR. If not, merge this first and then update it in another PR.